src/SDPoptimize.jl

```diff
@@ -182,9 +187,9 @@ Returns :
     of the system at each time step
 
 """
-function sdp_optimize(model::SPModel,
+function solve_DP(model::SPModel,
                   param::SDPparameters,
-                  display=true::Bool)
+                  display=true::Int64)
 
     SDPmodel = build_sdpmodel_from_spmodel(model::SPModel)
```

Comment:
This might take some time, maybe check with an if if model is already an SDPmodel ?